File: perl/src/tests/t_get_inittime.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$t = Atrshmlog::get_inittime_tsc_before();

print "get inittime : $t : \n";

$t = Atrshmlog::get_inittime_tsc_after();

print "get inittime : $t : \n";

($s,$n) = Atrshmlog::get_inittime();

print "get inittime : $s : $n : \n";

print " \n";

exit (0);

# end of main

File: python/src/tests/t_get_inittime.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

t = atrshmlog.get_inittime_tsc_before()

print('get inittime : ' + str(t) + ' : ')

t = atrshmlog.get_inittime_tsc_after()

print('get inittime : ' + str(t) + ' : ')

(s,n) = atrshmlog.get_inittime()

print('get inittime : ' + str(s) + ' : ' + str(n) + ' : ')

print (' ')

exit(0)

# end of test

